src/ucs/sys/sock.h

```diff
@@ -212,12 +213,14 @@ ucs_status_t ucs_socket_set_buffer_size(int fd, size_t sockopt_sndbuf,
  * @param [in]  socklen         Size of saddr.
  * @param [in]  backlog         Length of the queue for pending connections -
  *                              for the listen() call.
+ * @param [in]  ports_range     Range of ports from which a port will be set in saddr.
```

Comment:
it seems duplication that port number can be passed either by sockaddr or by ports_range, need to change the comment in line 210 and assert that saddr port number is always 0